README.md

```diff
@@ -71,15 +87,18 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+words = %w[Chip Potts Cogsworth Lumière Mrs. Potts]
```

Comment:
I would stick with the normal method of creating an array: `words = [ "Chip", "Potts", "Cogsworth" ]`